lib/socketclusterclient.rb

```diff
@@ -0,0 +1,263 @@
+require 'websocket-eventmachine-client'
+require 'json'
+require_relative './socketclusterclient/parser'
+require_relative './socketclusterclient/emitter'
+
+#
+# Class Socketcluster provides an interface to create a socket instance
+# use to connect to the socketcluster server
+# @author Maanav Shah <shahmaanav07@gmail.com>
+#
+class Socketcluster
```

Comment:
I think we can change the classname to ScClient, right now we are following the same name convention everywhere.

Reply:
Done.